packages/nuxt-locokit/src/runtime/stores/auth.ts

```diff
@@ -6,7 +6,9 @@ import { ref, useCookie } from '#imports'
 export const useStoreAuth = defineStore('auth', () => {
   const loading = ref(false)
   const isAuthenticated = ref(false)
-  const error = ref<Error | null>(null)
+  const error = ref<
+    Error | null | { code: string | null; content: string | null }
```

Comment:
What wlll contains "content"?

Reply:
An explanation of the error ?
Don't remember why I did this, but I think this is related to Feathers Errors.
I can remove it as, after checking the code, I can't see any reference to a `error.content`... WDYT ?

Reply:
Yep, same as "code" (for now :smile: ) and I am not sure if we will use it.